backend/workers/createRecurringEvents.js

```diff
@@ -140,26 +140,26 @@ module.exports = (cron, fetch) => {
                 body: jsonEvent
             }
 
-            console.log('Running createEvent: ', jsonEvent);
+            // console.log('Running createEvent: ', jsonEvent);
 
             try {
                 const response = await fetch(`${URL}/api/events/`, options);
                 const resJson = await response.json();
                 return resJson;
             } catch (error) {
-                console.log(error);
+                // console.log(error);
             };
         };
     };
 
     async function runTask() {
-        console.log("Creating today's events");
+        // console.log("Creating today's events");
 
         await fetchEvents();
         await fetchRecurringEvents();
         await filterAndCreateEvents();
 
-        console.log("Today's events are created");
+        // console.log("Today's events are created");
```

Comment:
My preference would be outright remove all of the `console.log()`s to keep things a bit more tidy but I'm open to discussion about it if you have a reason we should keep them as comments